LSE-61.tex

```diff
@@ -327,14 +327,20 @@ \subsubsection{Guider Calibration Data Acquisition}
 
 \subsection{Level 1 Data Products}
 
-\subsubsection{Nightly Data Accessible Within 24 hrs}
+\subsubsection{Nightly Data Accessible Within Specified Time}
 
 \label{DMS-REQ-0004}
 \textbf{ID:} DMS-REQ-0004 (Priority: 1b)
 
-\textbf{Specification:} With the exception of alerts and Solar System Objects, all Level 1 Data Products shall be made public within time \textbf{L1PublicT} (LSR-REQ-0104) of the acquisition of the raw image data. Alerts shall be made available within time \textbf{OTT1} (LSR-REQ-0101) from the conclusion of readout of the raw exposures used to generate each alert to the distribution of the alert to community distribution mechanisms.  Solar System Object orbits shall, on average, be calculated before the following night's observing finishes and the results shall be made available within time \textbf{L1PublicT} of those calculations being completed.
+\textbf{Specification:}
+With the exception of alerts and Solar System Objects, all Level 1 Data Products shall be made public within time \textbf{L1PublicT} of the acquisition of the raw image data.
+
+For visits resulting in fewer than \textbf{nAlertVisitPeak}, LSST shall be capable of supporting the distribution of at least \textbf{OTR1} per cent of alerts via the LSST alert distribution system within time \textbf{OTT1} from the conclusion of the camera's readout of the raw exposures used to generate each alert.
 
-\textbf{Discussion:} Because of the processing flow of SSObject orbit determination, meeting the base 24-hours-after-data-acquisition requirement would be far more challenging than for the other L1 Data Products, but the system throughput has to be good enough such that a back log can not build up.
+Solar System Object orbits will, on average, be calculated before the following night's observing finishes and the results \textit{shall} be made available within time \textbf{L1PublicT} of those calculations being completed.
+
+\textbf{Discussion:}
+    Because of the processing flow of SSObject orbit determination, meeting the base \textbf{L1PublicT}-hours-after-data-acquisition requirement would be far more challenging than for the other L1 Data Products, but the system throughput has to be good enough such that a back log can not build up.
```

Comment:
*L1PublicT* is a quantity with units.  "hours" should not appear after it.

Reply:
I've updated the model to remove the hours.